src/modules/app/services/waves/node/content/Aliases.js

```diff
@@ -63,9 +63,7 @@
                 // TODO : replace with waves-api method when it is implemented
                 return alias.length >= 4 &&
                     alias.length <= WavesApp.maxAliasLength &&
-                    alias.split('').some((char) => {
-                        return AVAILABLE_CHARS.indexOf(char) !== -1;
-                    });
+                    !alias.split('').some((char) => AVAILABLE_CHARS.indexOf(char) === -1);
```

Comment:
If I understand statement correctly, it might be replaced with following:
`alias.split('').every((char) => AVAILABLE_CHARS.includes(char));`.
Thus it is possible to get rid of explicit and implicit negation, implicit use of `includes` method.